src/scripts/h5p-multi-media-choice-option.js

```diff
@@ -261,6 +283,11 @@ export class MultiMediaChoiceOption {
         this.accessibilitySolutionText.parentNode.removeChild(this.accessibilitySolutionText);
       }
     }
+    if (this.feedbackText) {
+      if (this.feedbackText.parentNode) {
```

Comment:
This could be `if (this.feedbackText && this.feedbackText.parentNode) { ... }` (or even `if (this.feedbackText?.parentNode) { ... }` if Babel was told to transpile `optional chaining` as not all browsers's may support ES2020 yet.

Reply:
At first I just copied the style from the condition above but I will change it. And it looks like Babel transpiles chaining already without the need to explicitly specify it.

Reply:
@hpawe01 Really? [Indeed!](https://github.com/babel/babel/issues/10809) if `@babel/preset-env` is used. Oh, that's good to know.

Reply:
Ah, you are right, just had a look at `webpack.config.js` and overlooked `.babelrc`.